### PR TITLE
Streamline user management

### LINKED ISSUE #1337: Update 'USERS" tab of navbar to go directly to user management search component
### Overview
The `USERS` tab of the navbar initially shows two buttons that are no longer needed in our app. When a user clicks the `USERS` tab of the navbar, we want to immediately show them the search input to find a user.

### Action Items
- [x] Change the `UserAdmin` component so that it immediately shows the `UserManagement` component instead of the two buttons it is showing now
- [x] Remove the "Admin Dashboard" button from the `UserManagement` and `EditUsers` component

### Resources/Instructions
<details><summary>See where `USERS` tab directs to now</summary>

image

</details>

<details><summary>See where we want `USERS` tab direct</summary>

image

</details>

### PR DESCRIPTION
**Pull request by MattPereira**
Fixes #1337

### What changes did you make and why did you make them ?
  - Changed the `UserAdmin` component to immediately show the user managment search input
  - Removed the "Admin Dashboard" button because it was just showing users a redundant button
  - Removed confusing `PAGES` object logic from `UserAdmin` component 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

image

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

image

</details>

### CODE REVIEW

**Comment by MattPereira** [comment]
A backend unit test is failing but this PR does not touch a single file in the `backend/` folder :thinking:

**Comment by trillium** [comment]
Weird, magic removal of comment made backend tests pass again?

File: client/src/pages/UserAdmin.js [hunk 1]
```diff
@@ -7,20 +7,12 @@ import UserManagement from '../components/user-admin/UserManagement';
 import UserApiService from '../api/UserApiService';
 import ProjectApiService from '../api/ProjectApiService';
 
-const PAGES = Object.freeze({
```
Comment (Spiteless) [inline client/src/pages/UserAdmin.js:10]: Good riddance

Comment (trillium) [inline client/src/pages/UserAdmin.js:10]: Good riddance

File: client/src/pages/UserAdmin.js [hunk 2]
```diff
@@ -7,20 +7,12 @@ import UserManagement from '../components/user-admin/UserManagement';
...
-});
-
-// Parent
 const UserAdmin = () => {
   // Initialize state hooks
   const { auth } = useAuth();
   const [users, setUsers] = useState([]); // All users pulled from database
   const [projects, setProjects] = useState([]); // All projects pulled from db
   const [userToEdit, setUserToEdit] = useState({}); // The selected user that is being edited
-  const [currentPage, setCurrentPage] = useState(PAGES.main);
```
Comment (trillium) [inline client/src/pages/UserAdmin.js:23]: 🫡

**Review by trillium — APPROVED** [review]
Great job! I'm into these changes

**Pull request merged**